### DISCUSSION TITLE
How do I share a schematic with a friend

### DISCUSSION THREAD

**Discussion by StrangeHacks** [maruohon/litematica#295] [Q&A]
I play on an SMP with some of my friends and am wondering how I can share a schematic I made of an iron farm with him so we can be on the same page

**Comment by maruohon** [comment]
You just send them the schematic, and then tell them the placement origin coordinate and any possible rotation and mirror values.

In the 1.12.2 version there is also a feature that allows you to copy and paste the placement settings as a string, which allows easily sharing the origin point and any rotation values and also any possible sub-region position changes with other people (for example by copy pasting the settings string in Discord). But that feature hasn't yet been ported to the 1.13.2+ versions.

**Reply by Archeumaki** [reply]
can you put files in ?

**Reply by maruohon** [reply]
You put schematic files to the `schematics` directory inside the game directory.

See this for how to find it if you don't know where it is: https://old.reddit.com/r/litematica/comments/1co6vh4/downloading_schematics_on_different_versions_of/l3c21l9/

**Reply by Archeumaki** [reply]
tysm dude

**Reply by Archeumaki** [reply]
[dog farm wdl.zip](https://github.com/user-attachments/files/17017811/dog.farm.wdl.zip)
im trying to download this but it wont work thats just the problem but i know hoow to put other schematics inside now

**Reply by maruohon** [reply]
That's not a schematic, it's a world. You extract that to your saves directory.

**Reply by Archeumaki** [reply]
oh where is it in minecraft folder somewhere? lemme go check

**Reply by Archeumaki** [reply]
ye tysm works
